README.md

```diff
@@ -1,69 +1,273 @@
 # reply-urls-operator
-// TODO(user): Add simple overview of use/purpose
+A k8s Operator that keeps Ingress hosts in sync with the Redirect URLs associated with an Azure App Registration.
+
+## Table of Contents
+**[Description](#Description)**<br>
+**[How the Operator works](#How-the-Operator-works)**<br>
+**[Deploying the Operator to a cluster](#Deploying-the-Operator-to-a-cluster)**<br>
+**[Test out the operator locally](#Test-out-the-operator-locally)**<br>
+**[GitHub Workflows](#GitHub-Workflows)**<br>
+**[Modifying the API definitions](#Modifying-the-API-definitions)**<br>
+**[License](#License)**<br>
 
 ## Description
-// TODO(user): An in-depth paragraph about your project and overview of use
+The Reply URLs Operator was created to automate the manual process of updating and removing Redirect URLs when applications are created and removed from AKS clusters. This project aims to follow the Kubernetes [Operator pattern](https://kubernetes.io/docs/concepts/extend-kubernetes/operator/). It uses [Controllers](https://kubernetes.io/docs/concepts/architecture/controller/)
+which provides a reconcile function responsible for synchronizing resources until the desired state is reached on the cluster.
```

Comment:
documentation should be current tense in general, I think we can skip the manual information

Reply:
```suggestion
The Reply URLs Operator automates the manual process of updating and removing Redirect URLs when applications are created and removed from AKS clusters. This project aims to follow the Kubernetes [Operator pattern](https://kubernetes.io/docs/concepts/extend-kubernetes/operator/). It uses [Controllers](https://kubernetes.io/docs/concepts/architecture/controller/)
which provides a reconcile function responsible for synchronising resources until the desired state is reached on the cluster.
```